vtex-apps/category-menu
# Fix/incorrect align category items in mobile view

#### What is the purpose of this pull request?
Remove 'm15' class when treeLevel is equals to 3.

#### What problem is this solving?
It solves the wrong alignment in deeper categories. It occurs when the user is in mobile view. 

#### How should this be manually tested?
In mobile view, open the category menu, open "Decoration" and then "Smartphones". You should see "Accessories" and "Battery" as children from "See All" when actually they are in the same hierarchy.

#### Screenshots or example usage

Before
![captura de tela de 2019-01-15 11-33-47](https://user-images.githubusercontent.com/9946330/51186938-86237d80-18b9-11e9-9afd-7cf6feedf433.png)

After
![captura de tela de 2019-01-15 11-34-43](https://user-images.githubusercontent.com/9946330/51186990-a18e8880-18b9-11e9-9de7-3d472a3e6f6b.png)

[Workspace link](https://categorymenutreelevel--storecomponents.myvtex.com/)

#### Types of changes
